aldryn_client/localdev/main.py

```diff
@@ -199,21 +181,64 @@ def load_database_dump(client, path=None, recreate=False):
 
 
 def download_media(client, path=None):
+    click.secho('fetching media files', fg='green')
+
     path = path or os.path.join(utils.get_project_home(path), 'data/media')
     website_slug = utils.get_aldryn_project_settings(path)['slug']
-    click.secho('fetching media files', fg='green')
     backup_path = client.download_backup(website_slug)
+
     with tarfile.open(backup_path, 'r:gz') as backup_archive:
         media_archive_name = 'media_files.tar.gz'
         if media_archive_name not in backup_archive.getnames():
             click.secho('Media archive empty', fg='yellow')
             return
+
         media_fobj = backup_archive.extractfile(media_archive_name)
         with tarfile.open(fileobj=media_fobj, mode='r:gz') as media_archive:
             media_archive.extractall(path=path)
+
         click.secho('Downloaded media files into {}'.format(path), fg='green')
 
 
+def upload_database(client):
+    project_home = utils.get_project_home()
+    website_id = utils.get_aldryn_project_settings(project_home)['id']
+    dump_filename = 'local_dump.sql'
+    dump_filepath = os.path.join('/app/', dump_filename)
+    archive_path = os.path.join(project_home, 'local_db.tar.gz')
```

Comment:
since it's only temporary and less complicated to handle I think we're fine in ``project_root``